src/rubrix/server/commons/helpers.py

```diff
@@ -67,3 +67,36 @@ def unflatten_dict(
             d = d[part]
         d[parts[-1]] = value
     return resultDict
+
+
+def limit_value_length(data: Any, max_length: int) -> Any:
+    """
+    Given an input data, limits string values to a max_length by fetching
+    last max_length characters
+
+    Parameters
+    ----------
+    data:
+        Input data
+    max_length:
+        Max length for string values
+
+    Returns
+    -------
+        Limited version of data, if any
+    """
+
+    if isinstance(data, str):
+        return data[len(data) - max_length :]
+    if isinstance(data, dict):
+        return {
+            k: limit_value_length(v, max_length=max_length) for k, v in data.items()
+        }
+    if isinstance(data, (list, tuple, set)):
+        new_values = [limit_value_length(v, max_length=max_length) for v in data]
+        if isinstance(data, tuple):
+            return tuple(new_values)
+        if isinstance(data, set):
+            return set(new_values)
+        return new_values
```

Comment:
perhaps a bit more elegant
```suggestion
        new_values = map(lambda x: limit_value_length(x, max_length=max_length), data)
        return type(data)(new_values)
```

Reply:
¿list(_list) == _list?

Reply:
it looks like it is.

Reply:
hm, if you are referring to the return type of map, map returns an iterable, yielding the results.